packit/base_git.py

```diff
@@ -73,7 +73,11 @@ def create_branch(
         """
         # it's not an error if the branch already exists
         origin = self.local_project.git_repo.remote("origin")
-        head = self.local_project.git_repo.create_head(branch_name, commit=base)
+        try:
+            head = self.local_project.git_repo.create_head(branch_name, commit=base)
+        except OSError as ex:
+            logger.debug(f"It seems that branch {branch_name} already exists: {ex!r}.")
+            head = self.local_project.git_repo.branches[branch_name]
```

Comment:
What about `if branch_name in self.local_project.git_repo.branches`?

Reply:
else raise?

Reply:
I mean `if` instead of the `try` block...

Reply:
I meant: `if "what you said" else raise` :D

Reply:
What about this:
```python
            if branch_name in self.local_project.git_repo.branches:
                logger.debug(f"It seems that branch {branch_name} already exists: {ex!r}.")
                head = self.local_project.git_repo.branches[branch_name]
            else:
                head = self.local_project.git_repo.create_head(branch_name, commit=base)
```

What other error can occur? (Why we need the `try` block?)

Reply:
Sorry for nit-picking...;-)

Reply:
np, nit-picking implies better code

Reply:
and your suggestion made a ton of sense